blog-posts/event-sourcing-serveless/event-sourcing-serveless.md

```diff
@@ -0,0 +1,166 @@
+---
+published: false
+title: 'A prod-safe event sourcing architecture has never been so simple to implement in AWS serverless'
+cover_image:
+description: 'Description of the article'
+tags: serverless, eventsourcing, AWS, CQRS
+series:
+canonical_url:
+---
+
+If you're about to implement a **serverless** architecture for an event sourcing based application, you are in the right place. This article aims at sharing with you **2 years of learnings** :heart_eyes:, saving you the trouble to make the same mistakes I did, and providing you with a guidebook to have the most stable and the least painful solution you can go with. :rocket:
+
+> :books: In this article I assume that you have at least a basic knowledge of what is **CQRS**. [Here](https://docs.microsoft.com/en-us/azure/architecture/patterns/cqrs) is one excellent page to understand more deeply the ins and outs of CQRS pattern if you feel the need before diving into the interesting things.
+
+## Quick overview of an event sourcing solution implementing a CQRS interface
+
+:money_with_wings: Bank applications are one of the type of business requiring to **store a trace of all events** and **use the history** - just to name a few, accounting, medical records, transportation logistics...
```

Comment:
`use the history` c'est un peu bizarre, et je pense que ça ajoute pas grand chose. Par contre, précise que c'est des `business events`, sinon c'est pas clair.

Pareil pour `just to name a few`, la formulation est bizarre. Je retirerais cette partie.